ci/gitlab/.gitlab-ci.yml

```diff
@@ -3,7 +3,7 @@ stages:
 
 brew-lint-formula:
   stage: test
-  script: brew audit aktualizr.rb
+  script: brew audit --except-cops=Sorbet/FalseSigil,Style/Documentation,Style/FrozenStringLiteralComment,Metrics/MethodLength aktualizr.rb
```

Comment:
What's up with this stuff?

Reply:
works. I run it on CI
https://main.gitlab.in.here.com/olp/edge/ota/connect/client/homebrew-otaconnect/-/pipelines/1699818

Reply:
> What's up with this stuff?

seems build machine start using newer and strict version of rubocop during "brew audit" . I cannot add our own rubocop.yam so have to provide rules via CLI. e.g. "Metrics/MethodLength" it is check "functions longer than 10 lines are not allowed",
Style/FrozenStringLiteralComment - means always have "# frozen_string_literal: true" in comment block
Sorbet/FalseSigil -  same for "# typed: false/true"
Style/Documentation - always have comment block in the start oif rb file.

Reply:
> I cannot add our own rubocop.yam so have to provide rules via CLI

Why not?

At any rate, these seem like annoyances, so it's fine as is, I think.

Reply:
it ignoring it. because brew calls it indirectly maybe. Yes. it is just announcements.